src/routes/vocabulary-detail-exercise/VocabularyDetailExerciseScreen.tsx

```diff
@@ -54,31 +57,42 @@ const VocabularyDetailExerciseScreen = ({ route, navigation }: VocabularyDetailE
         labelOverride={labelOverrides?.closeExerciseHeaderLabel}
         isCloseButton={labelOverrides?.isCloseButton}
       />
-      <VocabularyDetail vocabularyItem={vocabularyItem} />
-      <Container>
-        <HorizontalLine />
+      <ScrollView>
+        <VocabularyDetail vocabularyItem={vocabularyItem} />
+        <Container>
+          <HorizontalLine />
 
-        <AlternativeWordsSection vocabularyItem={vocabularyItem} />
+          <AlternativeWordsSection vocabularyItem={vocabularyItem} />
 
-        <ButtonContainer>
-          {hasNextVocabularyItem ? (
-            <Button
-              label={getLabels().exercises.next}
-              iconRight={ArrowRightIcon}
-              onPress={goToNextWord}
-              buttonTheme={BUTTONS_THEME.contained}
-            />
-          ) : (
-            <Button label={closeExerciseLabel} onPress={navigation.goBack} buttonTheme={BUTTONS_THEME.contained} />
-          )}
-        </ButtonContainer>
-      </Container>
-      <FeedbackModal
-        visible={isFeedbackModalVisible}
-        onClose={() => setIsFeedbackModalVisible(false)}
-        feedbackType={FeedbackType.vocabularyItem}
-        feedbackForId={vocabularyItem.id}
-      />
+          <ButtonContainer>
+            {hasNextVocabularyItem && (
+              <Button
+                label={getLabels().exercises.next}
+                iconRight={ArrowRightIcon}
+                onPress={goToNextWord}
+                buttonTheme={BUTTONS_THEME.contained}
+              />
+            )}
+            {hasPreviousVocabularyItem && (
+              <Button
+                label={getLabels().exercises.previous}
+                iconLeft={ArrowLeftIcon}
+                onPress={goToPreviousWord}
+                buttonTheme={BUTTONS_THEME.text}
+              />
+            )}
+            {!hasNextVocabularyItem && (
+              <Button label={closeExerciseLabel} onPress={navigation.goBack} buttonTheme={BUTTONS_THEME.contained} />
```

Comment:
Why did you change this? Is it in the designs like that? I personally think that the placement of the previous word button should not change/switch with this one only on the last word. Instead, I would place the next word/close exercise always on top and the previous word button always on bottom.